src/test/java/org/cbio/gdcpipeline/tasklet/ProcessManifestFileTaskletTest.java

```diff
@@ -90,7 +90,7 @@ public void testCallGdcApiServiceUnavailable() throws Exception {
         httpHeaders.setContentType(MediaType.APPLICATION_JSON);
         HttpEntity<String> entity = new HttpEntity<String>("", httpHeaders);
         when(restTemplate.exchange(GDC_API_FILES_ENDPOINT, HttpMethod.POST, entity, String.class)).thenReturn(response);
-        tasklet.callGdcApi(GDC_API_FILES_ENDPOINT, "");
+        //tasklet.callGdcApi(GDC_API_FILES_ENDPOINT, "");
```

Comment:
is this for potentially fetching the files with graphql? if so, can you add a TO-DO explaining that this is the potential intended use for this tasklet call?

Reply:
I didn't really look at the tests too closely. I think this originally was for the REST api calls the pipeline did, which have been replaced with graphql calls. I'll check it out.